ui/widgets/nuxeo-uploader-behavior.js

```diff
@@ -476,6 +476,21 @@ export const UploaderBehavior = {
   _drop(e) {
     this.toggleClass('hover', false, this._dropZone);
     e.preventDefault();
-    this.uploadFiles(e.dataTransfer.files);
+    const allFiles = e.dataTransfer.items;
+    const files = [];
+    if (allFiles.length > 0) {
+      for (let i = 0; i < allFiles.length; i++) {
+        if (allFiles[i].webkitGetAsEntry().isDirectory) {
+          this.notify({ message: this.i18n('foderUpload.error') });
+        } else {
+          files.push(e.dataTransfer.files[i]);
+        }
+      }
+      if (files.length > 0) {
+        this.uploadFiles(files);
+      } else {
```

Comment:
Since the `_drop` method was not previously returning anything, I don't think you need this else condition.